packages/Table/src/components/ViewportBlock/ViewportBlock.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
```

Comment:
This component could made available externally or even its own package if other applications want to make use of it